src/pkg/member/manager.go

```diff
@@ -102,6 +104,10 @@ func (m *manager) DeleteMemberByUserID(ctx context.Context, uid int) error {
 	return m.dao.DeleteProjectMemberByUserID(ctx, uid)
 }
 
+func (m *manager) DeleteMemberByProjectID(ctx context.Context, projectID int64) error {
+	return m.dao.DeleteProjectMemberByProjectID(ctx, projectID)
```

Comment:
There is no implementation for this method.
Plus, it seems that this method has nothing to do with the "comma in username" issue. Maybe create a separate PR for `DeleteProjectMemberByProjectID`.

Reply:
removed